rohmu/object_storage/azure.py

```diff
@@ -113,6 +106,28 @@ def __init__(
         self.container = self.get_or_create_container(self.container_name)
         self.log.debug("AzureTransfer initialized, %r", self.container_name)
 
+    @staticmethod
+    def conn_string(
+        account_name: str,
+        account_key: Optional[str],
+        azure_cloud: Optional[str],
+        host: Optional[str],
+        port: Optional[int],
+        is_secure: bool,
+    ) -> str:
+        protocol = "https" if is_secure else "http"
+        conn = [
+            f"DefaultEndpointsProtocol={protocol}",
+            f"AccountName={account_name}",
+            f"AccountKey={account_key}",
+        ]
+        if not host and not port:
+            endpoint_suffix = AZURE_ENDPOINT_SUFFIXES[azure_cloud]
+            conn.append(f"EndpointSuffix={endpoint_suffix}")
+        else:
+            conn.append(f"BlobEndpoint={protocol}://{host}:{port}/{account_name}")
```

Comment:
So it is possible to create transfers bypassing the pydantic config models?

Reply:
I'm just saying that we are not indicating that `AzureTransfer` is private, so users could instantiate it directly instead of going through the factory methods that use `from_model` to create the instance from the pydantic configuration.

I don't this is a big deal, hence why I class this as nitpicking.